pkg/cmd/roachtest/github.go

```diff
@@ -286,11 +286,11 @@ func (g *githubIssues) createPostRequest(
 				"there should be a similar issue without the "+coverageLabel+" label. If there isn't one, it is "+
 				"possible that this failure is related to the code coverage infrastructure or overhead.")
 	}
-	if metamorphicBuild {
+	if runtimeAssertionsBuild {
 		topLevelNotes = append(topLevelNotes,
-			"This build has metamorphic test constants enabled. If the same failure was hit in a "+
-				"non-metamorphic run, there should be a similar issue without the "+metamorphicLabel+" label. If there "+
-				"isn't one, it is possible that this failure is caused by a metamorphic constant.")
+			"This build has runtime assertions enabled. If the same failure was hit in a run without assertions "+
+				"enabled, there should be a similar issue without the "+runtimeAssertionsLabel+" label. If there "+
+				"isn't one, it is possible that this failure hit an assertion or is related to the assertions overhead.")
```

Comment:
Nit: "...isn't one, then this failure is likely due to an assertion violation or (assertion) timeout."

Reply:
Done!